docs/develop/develop-overview.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Develop Office Add-ins
+description: An introduction to developing Office Add-ins.
+ms.date: 12/31/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins
+
+> [!TIP]
+> Please review [Building Office Add-ins](../overview/office-add-ins-fundamentals.md) before reading this article.
+
+All Office Add-ins are built upon the Office Add-ins platform and share a common framework through which certain capabilities can be implemented. For any add-in you build, you'll need to understand important concepts like host and platform availability, Office JavaScript API programming patterns, how to specify an add-in's settings and capabilities in the manifest file, and more. Core development concepts like these are covered here in the **Core concepts** > **Develop** section of the documentation. Review the information here before exploring the host-specific documentation that corresponds to the add-in you're building (for example, [Excel](../excel/index.md)).
```

Comment:
Consider breaking the first sentence into two: "All Office Add-ins are built upon the Office Add-ins platform. They share a common framework through which certain capabilities can be implemented."